README.md

```diff
@@ -19,6 +19,8 @@ The Flutter UI can be configured using a [YAML](https://yaml.org/) file.
     elevated-button-text-color: "#0000ff"
     window-title: "Custom Window Title"
 
+  live-session: true
```

Comment:
Do we want a completely separate config key for this, or should it be part of the `mode` enum?

Reply:
That's your decision... I'll put it wherever you prefer.

Reply:
In any case - could you rebase and merge the latest readme update @spydon landed in #431? I also think we should sync any changes to the whitelabel part in the readme with the [configuration guide](https://docs.google.com/document/d/10R0YOj4e8BTv6XPw9OE_y7GDy72xPqA5XP5lu0M7VbE/edit#heading=h.x8jmsnxxm1fh).

Reply:
I think this would fit well in the `mode` enum, we'll never have OEM core installations right?

Reply:
Well, I wouldn't say *never*, but I guess in that case we could add an `oem-core` mode :D

Reply:
Rebased and squashed (it wasn't trivial to rebase without it)

Reply:
Mmm... now I see what you mean, and I'm not sure, because using "oem-core" presumes that the only possibility for core desktop is to be distributed in a disk image without a live environment... And also that only core desktop will be distributed in a non-live disk image...

Reply:
Does core desktop expect user creation in stage 3, or is that supposed to be done later during first boot in stage 4? In the latter case we'd need to treat the user creation the same way we do in the oem case anyways.
In general, I wouldn't worry about possible other "modes" just yet. For the time being we care about:
* `standard` (Ubuntu Desktop + Flavors, live iso with user creation in stage 3 and no stage 4 at all)
* `oem` (additional eula page, no user creation in stage 3, init wizard with user creation on first boot aka stage 4)
* `core-desktop` (installation from image, no live session, ....?)

Reply:
Ok.

Reply:
Done.